### PR TITLE
Small investigation improvements

### LINKED ISSUE #264: Add an investigator column in investigations list
### Description
If you work in a team of investigators, you can't distinguish between yours and others' investigations.
Maybe adding a column of investigator, and being able to also seeing all investigations, can do the work.

#### Expected behavior
In List of investigations, add a column of the investigator, and an option to see all investigations also (for admins).

#### Actual behavior
You can see all investigations, without filtering to yours/others only.

### PR DESCRIPTION
**Pull request by gaelmuller**
Some improvements on investigations:

  - Add a new investigation button in the "New" menu
  - List the username that created the investigation (fixes #264 and #60)
  - Show "Unnamed" instead of "None"

### CODE REVIEW

File: core/investigation.py [hunk 1]
```diff
@@ -34,14 +37,20 @@ class InvestigationEvent(EmbeddedDocument):
     nodes = ListField(ReferenceField('Node'))
     datetime = DateTimeField(default=datetime.utcnow)
 
+def default_user():
+    try:
+        return current_user.username
+    except:
```
Comment (tomchop) [inline core/investigation.py]: Can you be a bit more specific here?

File: core/web/api/templates/investigation_api.html [hunk 2]
```diff
@@ -8,16 +8,26 @@
...
+        {% if obj['name'] %}
+          {{obj['name']}}
+        {% else %}
+          Unnamed
+        {% endif %}
+      </a>
+    </td>
     <td>{{macros.display_datetime(obj.created)}}</td>
     <td>{{macros.display_datetime(obj.updated)}}</td>
+    <td>{% if obj.created_by %}{{obj.created_by}}{% endif %}</td>
```
Comment (tomchop) [inline core/web/api/templates/investigation_api.html]: What about something like `{{obj.created_by or ''}}`. This allows you to remove the `{%if%}` clause

File: core/web/frontend/templates/investigation/single.html [hunk 3]
```diff
@@ -23,7 +23,11 @@ <h4 class="panel-title"></h4>
         <div class="panel panel-default">
           <div class="panel-heading">
             <h3 class="panel-title">
-                {{investigation.name}}
+                {% if investigation.name %}
```
Comment (tomchop) [inline core/web/frontend/templates/investigation/single.html]: `{{investigation.name or 'Unnamed Investigation'}}`

File: core/investigation.py (hunk 1 above)
Comment (gaelmuller) [reply core/investigation.py]: It was not needed, removed it

**Comment by haxpak** [comment]
need another improvement:

1. add existing entities to investigation. there is only an option to add new entities, however there can be an old investigation / malware that can relate to a new investigation

2. export graph of investigation. investigation graph may be exported to a PDF format / any other suitable format that can be presented to the bosses

**Comment by gaelmuller** [comment]
Could you please create issues for these requests ? This PR has been merged, so it is no longer tracked.

1. Correct, this is missing
2. FYI in Chrome you can right-click and save the graph as an image

**Pull request merged**